### ISSUE TITLE
doc: The 'ublue-os/fleek' repository doesn't contain the 'internal/nix/home.nix.tmpl' path in 'main'.  

### ISSUE THREAD

**Issue by iennae** [ublue-os/fleek#137]
Readme includes a link to where the templates are getting pulled from that reports 

```
The 'ublue-os/fleek' repository doesn't contain the 'internal/nix/home.nix.tmpl' path in 'main'.  
```

I looked for the templates to see if I could submit a PR but they don't appear to be in this repo.

**Comment by bketelsen** [comment]
thanks for the report!

The readme is fixed in this pr: #138 

the templates are in /internal/flake/*.nix.tmpl while you wait for that to get approved.

**Comment by bketelsen** [comment]
fixed by #138
